CRM/ACL/BAO/ACL.php

```diff
@@ -525,12 +525,23 @@ private static function getGroupClause(array $groupIDs, string $operation): stri
     }
 
     if ($groupIDs) {
```

Comment:
This function is funny. It requires an array of `$groupIDs` and executes a qurey that will definitely crash if the array is empty (L511 `WHERE g.id IN ( $ids )`) but then this conditional exists. If we wanted it to do something we could move it before the query, but if it hasn't crashed yet then it's probably safe to assume it's always true and just remove it.